roles/ovirt-aaa-jdbc/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
 ## User & groups internal
 ################################
 - name: Manage internal users
-  command: "{{ aaa_jdbc_prefix }}/ovirt-aaa-jdbc-tool user {{ (item.state is undefined or item.state == 'present') | ternary('add','delete') }} {{ item.name }}"
+  command: "{{ aaa_jdbc_prefix }}/ovirt-aaa-jdbc-tool user {{ (item.state is undefined or item.state == 'present') | ternary('add','delete') }} {{ item.name }} {% if item['attributes'] is defined %} {% for attr, value in item['attributes'].iteritems() %} --attribute={{ attr }}={{ value }} {% endfor %} {% endif %}"
```

Comment:
{{ value }} maybe value should be surrounded by quotas to allow values with spaces?

Reply:
will fix.